observations/__init__.py

```diff
@@ -19,6 +19,7 @@
 from observations.cifar10 import cifar10
 from observations.cifar100 import cifar100
 from observations.crabs import crabs
+from observations.download_utils import download_file
```

Comment:
We probably don't need to expose `download_file` to users if we have `maybe_download_and_extract`.